pkg/operator/staticpod/controller/revision/revision_controller.go

```diff
@@ -193,7 +193,7 @@ func (c RevisionController) isLatestRevisionCurrent(revision int32) (bool, strin
 		}
 		if !equality.Semantic.DeepEqual(existingData, requiredData) {
 			if klog.V(4) {
-				klog.Infof("secret %q changes for revision %d: %s", s.Name, revision, resourceapply.JSONPatch(existing, required))
+				klog.Infof("Secret %q changes for revision %d: %s", s.Name, revision, resourceapply.JSONPatchSecret(existing, required))
```

Comment:
preexisting but it si missing namespace

Reply:
`RevisionResource` does not hold Namespace, thus we don't print it.